trlc/ast.py

```diff
@@ -1074,6 +1074,7 @@ def evaluate(self, mh, context):
         return Value(self.location, self, self.typ)
 
     def resolve_references(self, mh):
+        # lobster-trace: LRM.Markup_String_Late_Reference_Resolution
```

Comment:
move to https://github.com/bmw-software-engineering/trlc/blob/92947777b0d034c37b2db1fb82502c420ca50326/trlc/trlc.py#L507 instead (function `resolve_record_references` in `trlc.py`)